frontend/pages/SoftwarePage/SoftwareOSDetailsPage/_styles.scss

```diff
@@ -4,6 +4,10 @@
   flex-direction: column;
   gap: $pad-medium;
 
+  .team-dropdown-wrapper {
```

Comment:
These one's without the 4 lines of SCSS were looking pretty good except the missing `height: 38px` made the gap from the dropdown and the first component larger than it should be. The other 3 properties included in `@include normalize-team-header` don't do much to this dropdown wrapper HOWEVER, if we add buttons on the right side of these pages like the other pages, it would already be styled correctly with display flex, align items center and justify content space between.

@jmwatts I think that's it on this ticket, hope this helps QA!